CHANGELOG.md

```diff
@@ -1,3 +1,37 @@
+## 1.19.0
+
+This release contains multiple changes to rkt core, bringing it more in line with the new Container Runtime Interface (CRI) from Kubernetes.
+
+A new experimental `app` subcommand has been introduced, which allows creating a "pod sandbox" and dynamically mutating it at runtime. This feature is not yet completely stabilized, and is currently gated behind an experimental flag.
+
+### New features and UX changes
+- rkt: experimental support for pod sandbox ([#3318](https://github.com/coreos/rkt/pull/3318)). This PR introduces an experimental `app` subcommand and many additional app-level options.
+- rkt/image: align image selection behavior for the rm subcommand ([#3353](https://github.com/coreos/rkt/pull/3353)).
+- stage1/init: leave privileged pods without stage2 mount-ns ([#3290](https://github.com/coreos/rkt/pull/3290)).
+- stage0/image: list images output in JSON format ([#3334](https://github.com/coreos/rkt/pull/3334)).
+- stage0/arch: initial support for ppc64le platform ([#3315](https://github.com/coreos/rkt/pull/3315)).
+
+### Bug fixes:
+- gc: make sure `CNI_PATH` is same for gc and init ([#3348](https://github.com/coreos/rkt/pull/3348)).
+- gc: clean up some GC leaks ([#3317](https://github.com/coreos/rkt/pull/3317)).
+- stage0: minor wording fixes ([#3351](https://github.com/coreos/rkt/pull/3351)).
+- setup-data-dir.sh: fallback to the `mkdir/chmod`s if the rkt.conf doesn't exist ([#3335](https://github.com/coreos/rkt/pull/3335)).
+- scripts: add gpg to Debian dependencies ([#3339](https://github.com/coreos/rkt/pull/3339)).
+- kvm: fix for breaking change in Debian Sid GCC default options ([#3354](https://github.com/coreos/rkt/pull/3354)).
+- image/list: bring back field filtering in plaintext mode ([#3361](https://github.com/coreos/rkt/pull/3361)).
+
+### Other changes
+- cgroup/v1: introduce mount flags to mountFsRO ([#3350](https://github.com/coreos/rkt/pull/3350)).
+- kvm: update QEMU version to 2.7.0 ([#3341](https://github.com/coreos/rkt/pull/3341)).
+- kvm: bump kernel version to 4.8.6, updated config ([#3342](https://github.com/coreos/rkt/pull/3342)). 
+- vendor: introduce kr/pretty and bump go-systemd ([#3333](https://github.com/coreos/rkt/pull/3333)).
+- vendor: update docker2aci to 0.14.0 ([#3356](https://github.com/coreos/rkt/pull/3356)).
+- docs/networking: minor clarification to DHCP type ([#3325](https://github.com/coreos/rkt/pull/3325)).
+- docs: explain how to use rkt-builder to build rkt ([#3331](https://github.com/coreos/rkt/pull/3331)).
+- tests: add the --debug option to more tests ([#3340](https://github.com/coreos/rkt/pull/3340)).
+- scripts/build-rir: bump rkt-builder version to 1.1.1 ([#3360](https://github.com/coreos/rkt/pull/3360)).
+- docs: update to reference-style markdown links ([#3321](https://github.com/coreos/rkt/pull/3321)).
```

Comment:
We could consolidate `docs` changes under one bullet point ` Documentation updates: `, as we did in the previous releases.